### PR TITLE
Fix/end session

### PR DESCRIPTION
**Pull request by m-abs**

### CODE REVIEW

File: src/models/member/session.coffee [hunk 1]
```diff
@@ -21,6 +21,10 @@ LYT.session = do ->
       if credentials.status is 'ok'
         log.message 'Session: init: reading credentials from getNotaAuthToken()'
         LYT.session.setCredentials credentials.username, credentials.token
+        LYT.service.logOn credentials.username, credentials.token
```
Comment (ddfreiling) [inline src/models/member/session.coffee]: What if this `logOn` fails?

File: package.json [hunk 2]
```diff
@@ -3,12 +3,18 @@
...
+    "build:dev": "npx cake -d app",
+    "build": "npx cake -cmt app"
+  },
   "directories": {
     "doc": "doc"
   },
   "devDependencies": {
     "body-parser": "^1.8.4",
     "coffeelint": "^2.1.0",
+    "coffeescript": "^1.11.1",
```
Comment (ddfreiling) [inline package.json]: should we lock this? Alternatively we could check-in package-lock.json, so that nota.dk can `npm ci` in their build script

**Comment by ddfreiling** [comment]
Last time I tried to run it with `https://` I got mixed content warnings on icon and image resources as well. Is this fixed? 🤔

File: src/models/member/session.coffee (hunk 1 above)
Comment (m-abs) [reply src/models/member/session.coffee]: Should trigger an error that redirects to nota.dk

File: src/config/config.dev.coffee [hunk 3]
```diff
@@ -20,3 +20,35 @@ jQuery.extend LYT.config,
...
+    # process fails _unexpectedly_, i.e. connection
+    # drops etc.) . If the server explicitly rejects
+    # the username/password, `service` won't stupidly
+    # try the same combo X more times.
+    logOnAttempts: 3 # default: 3
+    guestUser: "guest"
+    guestLogin: "guest"
+
+    externalLogin:
+      redirectOnNoBook: true
```
Comment (ddfreiling) [inline src/config/config.dev.coffee]: @m-abs  This config is nested under `externalLogin`, but assumed to be under `service` in check below

Comment (m-abs) [reply src/config/config.dev.coffee]: Good catch.

**Comment by m-abs** [comment]
`https://` should be fixed for icons.

**Pull request merged**